internal/server/place/golden/get_place_stat_date_within_place_test.go

```diff
@@ -27,57 +27,65 @@ import (
 )
 
 func TestGetPlaceStatDateWithinPlace(t *testing.T) {
+	t.Parallel()
 	ctx := context.Background()
-
-	client, _, err := e2e.Setup()
-	if err != nil {
-		t.Fatalf("Failed to set up mixer and client")
-	}
-	_, filename, _, _ := runtime.Caller(0)
-	goldenPath := path.Join(
-		path.Dir(filename), "get_place_stat_date_within_place")
-
-	for _, c := range []struct {
-		ancestorPlace string
-		placeType     string
-		statVars      []string
-		goldenFile    string
-	}{
-		{
-			"geoId/06",
-			"County",
-			[]string{"Count_Person", "Median_Age_Person"},
-			"CA_County.json",
-		},
-		{
-			"country/USA",
-			"State",
-			[]string{"Count_Person", "Count_Person_Female"},
-			"USA_State.json",
-		},
+	for _, opt := range []*e2e.TestOption{
+		{UseCache: true, UseMemdb: true},
+		{UseCache: true, UseMemdb: true, UseImportGroup: true},
 	} {
-		resp, err := client.GetPlaceStatDateWithinPlace(ctx, &pb.GetPlaceStatDateWithinPlaceRequest{
-			AncestorPlace: c.ancestorPlace,
-			PlaceType:     c.placeType,
-			StatVars:      c.statVars,
-		})
+		client, _, err := e2e.Setup()
```

Comment:
need to pass in opt here "e2e.Setup(opt)"

Reply:
thanks for the catch! fixed. oddly, no change in the results ...

Reply:
That's good, so there are no diff between current and IG!